app/src/androidTest/java/com/urbandroid/sleep/addon/stats/model/socialjetlag/SocialJetlagStatsTest.java

```diff
@@ -71,4 +76,169 @@ public void testGetRecordIrregularity() {
 
         assertThat(stats.getRecordIrregularity(record)).isEqualTo( ((6.625f - 3.5f) + (8.5f - 6.5f)) / 2f );
     }
+
+    @Test
+    public void calculateSRIEdgeCases() {
+        SocialJetlagStats  stats = new SocialJetlagStats(cr, false);
+
+        // one bitset empty
+        BitSet day1Interval = new BitSet(1440);
+        BitSet day2Interval = new BitSet();
+        day1Interval.set(0,720);
+        Assert.assertEquals(1.0, stats.calculateSRI(day1Interval, day2Interval), .0001);
```

Comment:
what's the 0.0001 for? 

hmm .. maybe we should change the expected behavior - if one of them is empty we should probably just throw an error

Reply:
I believe you need to set a threshold for how similar two floats are when using assertEquals on floats